packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -246,11 +246,14 @@ class SnackBar extends StatefulWidget {
     this.onVisible,
     this.dismissDirection = DismissDirection.down,
     this.clipBehavior = Clip.hardEdge,
+    this.actionOverflowThreshold,
```

Comment:
nit: consider reordering such that this property is directly after the `action` property